flaml/automl/automl.py

```diff
@@ -490,6 +490,16 @@ def save_best_config(self, filename):
         with open(filename, "w") as f:
             json.dump(best, f)
 
+    @property
+    def feature_transformer(self):
+        """Returns AutoML Transformer"""
+        return self._transformer
+
+    @property
+    def label_transformer(self):
+        """Returns AutoML label transformer"""
+        return self._label_transformer
```

Comment:
```suggestion
        return getattr(self, "_label_transformer", None)
```
In case the attribute is not set.